tests/tasks/setup/eks/awscli-cp-with-vpc.yaml

```diff
@@ -45,7 +45,11 @@ spec:
       if [ "$CREATED_CLUSTER" == "" ]; then
         aws eks create-cluster --name $(params.cluster-name) --region $(params.region) --kubernetes-version $(params.kubernetes-version) --role-arn $(params.servicerole) --resources-vpc-config subnetIds=$subnets,securityGroupIds=$sg $ENDPOINT_FLAG
       fi
-      aws eks $ENDPOINT_FLAG --region $(params.region) wait cluster-active --name $(params.cluster-name) 
+      date
+      aws eks $ENDPOINT_FLAG --region $(params.region) wait cluster-active --name $(params.cluster-name)
+      date 
       aws eks $ENDPOINT_FLAG update-kubeconfig --name $(params.cluster-name) --region $(params.region)
       # enable PD on the cluster 
-      kubectl set env daemonset/aws-node -n kube-system ENABLE_PREFIX_DELEGATION=true
+      # To do - remove this, temporarily adding because apparently apiserver is not recongnizing deamonset for sometime inconsistently, need to see if livez and readyz are active
+      sleep 300 
```

Comment:
Does it really need 5 more minutes to install the APIs?
I'd suggest generating some debug info before `sleep`:
```
kubectl api-versions
kubectl api-resources
```
We can understand what APIs are installed.